python/ray/dag/compiled_dag_node.py

```diff
@@ -426,7 +473,7 @@ def _compute(self, class_handle) -> bool:
 
         resolved_inputs = []
         for task_input in self.task_inputs:
-            resolved_inputs.append(task_input.resolve(input_data))
+            resolved_inputs.append(task_input.resolve(input_data, cache))
```

Comment:
can you rename cache -> input_result_cache

Reply:
Code is refactored and this is gone.